p2p/net/connmgr/connmgr.go

```diff
@@ -375,7 +375,7 @@ func (cm *BasicConnMgr) doTrim() {
 func (cm *BasicConnMgr) trim() {
 	// do the actual trim.
 	for _, c := range cm.getConnsToClose() {
-		log.Infow("closing conn", "peer", c.RemotePeer())
+		log.Debugw("closing conn", "peer", c.RemotePeer())
```

Comment:
This one is fine.

Reply:
Are you sure it's debug? Look at this:
```
admin@node-01.sm-eu-zur1.spiff.app-test:~ % ls -lh /var/log/docker/spiff-workflow-wakunode/docker.log
-rw-r--r-- 1 syslog syslog 13G Mar  2 13:13 /var/log/docker/spiff-workflow-wakunode/docker.log

admin@node-01.sm-eu-zur1.spiff.app-test:~ % grep 'closing conn' /var/log/docker/spiff-workflow-wakunode/docker.log | wc -l
57729782
```
This is from 1.5 day of not really that active of a node. If you have a trace level it should be trace.

Reply:
I don't think we have a trace log level. Or if we do, we don't use it.

Reply:
Fair enough.